brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
                          sizeof(float));
-
-                sectionsMapping.insert(
+                sectionsMapping.push_back(
                     std::make_pair(current, std::make_pair(frameIndex, 0)));
 
                 if (previous != LB_UNDEFINED_UINT16)
-                    sectionsMapping[previous].second = count;
+                    sectionsMapping[sectionsMapping.size() - 2].second.second =
```

Comment:
Then this code is never executed. This is already the case of soma reports and the tests pass :)
Note that this line would only be executed if previous = undefined and current != previous. In that case the push back in line 722 has already been executed twice.  You could argue that 730 is actually wrong if the cell has no compartments at all, but if that was the case, that would mean there's something reaaaly broken somewhere else.